modules/installation-vsphere-installer-infra-requirements.adoc

```diff
@@ -40,6 +40,19 @@ A user requires the following privileges to install an {product-title} cluster:
 
 For more information about creating an account with only the required privileges, see link:https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.security.doc/GUID-5372F580-5C23-4E9C-8A4E-EF1B4DD9033E.html[vSphere Permissions and User Management Tasks] in the vSphere documentation.
 
+[discrete]
+[id="installation-vsphere-installer-infra-requirements-vmotion_{context}"]
+== Using {product-title} with vMotion
+
+[IMPORTANT]
+====
+{product-title} generally supports compute-only vMotion. Storage vMotion is not supported.
+====
+
+If you are using vSphere volumes in your pods, migrating a VM across datastores either manually or through Storage vMotion causes invalid references within {product-title} persistent volume (PV) objects. This prevents affected pods from starting up and can result in data loss. Therefore, any form of Storage vMotion is unsupported.
```

Comment:
We may not want to use the statement 'any form' (its fairly strong wording). 
In theory, compute/memory migration with vmotions work just fine. 

We may want to discuss this with Maria Bracho <mbracho@redhat.com> on what wording the BU would like to see/user here with this provider. 

cc: @listey

Reply:
LGTM

Reply:
I reached out to mbracho@redhat.com by email for ack on this but haven't heard back yet. @sferich888 - because @listey has approved for PM, can you ack for PX? Or do you still want to wait for Maria for ack from VMware installer perspective?